src/ui/public/visualize/loader/types.ts

```diff
@@ -28,7 +28,7 @@ export interface TimeRange {
 }
 
 export interface Filter {
-  meta: object;
+  meta: any;
```

Comment:
just curious - why does this type definition need to be relaxed?

Reply:
else tslint was throwing some errors about properties being non existent on object.